### PR TITLE
(chore): Migrate CI to GitHub Actions

### PR DESCRIPTION
**Pull request by rashil2000**
Relates to https://github.com/ScoopInstaller/Main/pull/3361

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Pinging @r15ch13 to disable the AppVeyor pipeline

**Comment by niheaven** [comment]
`#Requires -Modules @{ ModuleName = 'Pester'; MaximumVersion = '4.99' }` should be added to `test.ps1`

**Comment by rashil2000** [comment]
It wasn't present before though. Why is it necessary?

**Comment by niheaven** [comment]
As my previous comment, all scoop tests don't support Pester 5 which is preinstalled in GitHub's Windows image. If we don't add this line, GitHub Actions will try to use Pester 5 even if `init.ps1` in scoop core installed version 4. Appveyor doesn't has Pester installed so it will use version 4 that installed by `init.ps1`.

**Comment by rashil2000** [comment]
Oh, didn't know that. Added it 👍

File: bin/test.ps1 [hunk 1]
```diff
@@ -1,2 +1,5 @@
+#Requires -Modules @{ ModuleName = 'Pester'; MaximumVersion = '4.99' }
+
 if(!$env:SCOOP_HOME) { $env:SCOOP_HOME = Resolve-Path (scoop prefix scoop) }
```
Comment (niheaven) [inline bin/test.ps1:3]:
```suggestion
if(!$env:SCOOP_HOME) { $env:SCOOP_HOME = Convert-Path (scoop prefix scoop) }
```

Comment (niheaven) [reply bin/test.ps1:3]: > Resolves the wildcard characters in a path, and displays the path contents.

See [Resolve-Path](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/resolve-path?view=powershell-5.1)

For most usage, we should use `Convert-Path` which just `Converts a path from a PowerShell path to a PowerShell provider path.` unless using `Resolve-Path -Relative`.

**Review by niheaven — CHANGES_REQUESTED** [review]

File: bin/test.ps1 (hunk 1 above)
Comment (niheaven) [reply bin/test.ps1:3]: Forgotten to change them in `Main` bucket...

**Review by niheaven — APPROVED** [review]

**Pull request merged**